lib/actions.js

```diff
@@ -35,30 +35,39 @@ var ACCOUNT_STATUS_ACTION = {
 
 // Actions that send an email, and hence might make
 // us look like spammers if abused.
-var EMAIL_SENDING_ACTION = {
+const EMAIL_SENDING_ACTION = {
   accountCreate: true,
   recoveryEmailResendCode: true,
   passwordForgotSendCode: true,
   passwordForgotResendCode: true,
   sendUnblockCode: true
 }
 
+// Actions that can send sms, and could make us
+// very annoying to a user if abused.
+const SMS_SENDING_ACTION = {
+  inviteUserSms: true
```

Comment:
@vladikoff Thought about it some more and really what is happening is the user is sending an sms to connect a device, not inviting a user. Renamed to `connectDeviceSms`.